04homework/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace Homework4
+{
+    using System;
+
+    class Homework4
+    {
+
+        static int MaxValue(params int[] numbers)
+        {
+            return numbers.Max();
+        }
+
+        static int MinValue(params int[] numbers)
+        {
+            return numbers.Min();
+        }
+
+        static int TrySumIfOdd(int x, int y, out bool isOdd)
```

Comment:
this method should return bool, not int, use Try pattern
more about it: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/exceptions-and-performance#try-parse-pattern